pyproject.toml

```diff
@@ -63,10 +63,13 @@ name = "Ivan Carvalho"
 email = "ivancarvalho@gatech.edu"
 
 [build-system]
-requires = ["setuptools>=70.1.0", "setuptools-rust>=1.9"]
+requires = ["setuptools>=77.0.0", "setuptools-rust>=1.9"]
```

Comment:
This is something I overlooked on #1420: because of https://github.com/pypa/packaging-problems/issues/870, we need a newer version of setuptools.

Nevertheless, `pip` should handle this smoothly.